Header header
float64 latitude
float64 longitude
float64 altitude
float64 utm_easting
float64 utm_northing
uint8 zone
string letter
uint8 fix_quality
float64 hdop
string gngga_read
